bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,289 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affects_comments: bool=True,
+                        space_before_and_after_arrow: bool=False,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=0,
+                        spacing_in_empty_braces: int=0,
+                        camelcase_classes_name: bool=True,
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        enforce_newline_at_EOF: bool=False,
+                        use_spaces: bool=True,
+                        tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                        newlines_after_classes: bool=False,
+                        number_of_newlines_after_classes: int=3,
+                        no_backticks: bool=True,
+                        no_implicit_braces: bool=False,
+                        no_implicit_parentheses: bool=False,
+                        no_interpolation_in_single_quotes: bool=True,
+                        no_stand_alone_at: bool=True,
+                        disable_throwing_strings: bool=True,
+                        no_trailing_semicolons: bool=True,
+                        no_trailing_whitespaces: bool=True,
```

Comment:
should be allow_trailing_whitespace as per https://github.com/coala-analyzer/bear-docs/blob/master/docs/SpaceConsistencyBear.rst, they mean the same